$NetBSD: patch-Makefile.inc.in,v 1.2 2015/03/24 20:27:49 pebog Exp $

Don't do any ownership changes when NO_SYS_INSTALL is set.  Some of
these variables (BINOWN, BINGRP, MANOWN, MANGRP..) will later be passed
to bsd.*.mk include files and cause failure during unprivileged builds.

Also, NO_SYS_INSTALL shouldn't prevent man pages from being installed
(NOMAN=no).

--- Makefile.inc.in.orig	2014-08-06 14:43:42.000000000 +0000
+++ Makefile.inc.in
@@ -95,6 +95,7 @@ BINDIR	    =$(DCC_BINDIR)
 BINDIR	    =@installroot@@bindir@
 @MAKE_DOT@endif
 
+@MAKE_DOT@ifndef NO_SYS_INSTALL
 @MAKE_DOT@ifndef DCC_OWN
 @MAKE_DOT@ifdef NO_SUID
 DCC_OWN	    =@DCC_OWN@
@@ -103,11 +104,14 @@ DCC_OWN	    =@DCCSUID@
 @MAKE_DOT@endif
 @MAKE_DOT@endif
 BINOWN	    =$(DCC_OWN)
+@MAKE_DOT@endif
 
+@MAKE_DOT@ifndef NO_SYS_INSTALL
 @MAKE_DOT@ifndef DCC_GRP
 DCC_GRP	    =@DCC_GRP@
 @MAKE_DOT@endif
 BINGRP	    =$(DCC_GRP)
+@MAKE_DOT@endif
 
 @MAKE_DOT@ifndef DCC_MODE
 DCC_MODE    =555
@@ -115,16 +119,17 @@ DCC_MODE    =555
 BINMODE	    =$(DCC_MODE)
 
 MANDIR	    =@installroot@@mandir@
+@MAKE_DOT@ifndef NO_SYS_INSTALL
 @MAKE_DOT@ifndef MANGRP
 MANGRP	    =$(DCC_GRP)
 @MAKE_DOT@endif
 @MAKE_DOT@ifndef MANOWN
 MANOWN	    =$(DCC_OWN)
 @MAKE_DOT@endif
+@MAKE_DOT@endif
 MANMODE	    =444
 
 @MAKE_DOT@ifdef NO_SYS_INSTALL
-NOMAN	=no
 SET_BINOWN=
 SET_MANOWN=
 SET_DCCOWN=
